src/components/FeeFineDetailsButton/FeeFineDetailsButton.js

```diff
@@ -15,6 +15,7 @@ const FEE_FINES_STATUSES = {
 
 class FeeFineDetailsButton extends React.Component {
   static propTypes = {
+    isVirtualUser: PropTypes.bool,
```

Comment:
Could you please clarify should we add `isRequired `or add default value?

Reply:
added default prop value